dashboard/db/migrate/20161104075248_add_first_and_last_names_to_pd_enrollments.rb

```diff
@@ -0,0 +1,30 @@
+class AddFirstAndLastNamesToPdEnrollments < ActiveRecord::Migration[5.0]
+  class Pd::Enrollment < ActiveRecord::Base
+  end
+
+  def change
+    change_table(:pd_enrollments) do |t|
+      t.string :first_name
+      t.string :last_name
+    end
+
+    # Migrate the existing data: Split each (full) name on the first space.
+    reversible do |dir|
+      dir.up do
+        Pd::Enrollment.find_each do |enrollment|
+          name_split = enrollment.name.split(' ', 2)
+          enrollment.first_name = name_split[0]
+          enrollment.last_name = name_split[1] || ''
+          enrollment.save!
+        end
+      end
+    end
```

Comment:
Do we have some guarantee that this migration will never get re-run? If not, it might be nice (and easy) to only populate first and last name from `name` if first and last name are not already set.